modules/01-mods.js

```diff
@@ -304,6 +304,31 @@ async function slashModTrust(interaction) {
   return interaction.editReply(trust);
 }
 
+/** @param {Augur.GuildInteraction<"CommandSlash">} interaction*/
+async function slashModTrustAudit(interaction) {
+  try {
+    await interaction.deferReply({ flags: u.ephemeralChannel(interaction, u.sf.channels.mods.discussion) });
+    const threshold = interaction.options.getInteger("posts", false) || 100;
```

Comment:
We should probably use ?? instead of ||

?? Provides an alternate value if the given value is undefined or null, but || provides an alternate value if the given value is falsey (0, "", false, undefined, and null are all 'falsey' values)